addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -448,6 +371,185 @@
     </section>
 </template>
 
+<!-- Job Application Form -->
+<template id="apply">
+    <t t-call="website.layout">
+        <!-- Topbar -->
+        <t t-set="o_jobs_topbar_title">
+            <a t-attf-href="/jobs/detail/#{job.id}">
+                <i class="fa fa-long-arrow-left text-primary me-2"/><span t-field="job.name"/>
+            </a>
+        </t>
+        <t t-call="website_hr_recruitment.topbar"/>
+        <!-- Content -->
+        <div id="wrap"  class="container">
+            <!-- Title -->
+            <div t-if="job" class="row pt24">
+                <div class="col-lg-8">
+                    <h1 class="text-left">
+                        <span t-field="job.name"/>
+                    </h1>
+                    <h5 class="fw-light text-left mt-2 text-muted">
+                        Job Application Form
+                    </h5>
```

Comment:
With BS5, aren't we supposed to use start & end instead of left & right ? (also happens in other blocks in this file)

Reply:
Good one! I just fixed it, thanks! 👍